neo/rawio/axonarawio.py

```diff
@@ -123,15 +123,16 @@ def _parse_header(self):
         params['set']['file_header'] = set_dict
         params['set']['sampling_rate'] = int(set_dict['rawRate'])
 
+        signal_streams = self._get_signal_streams_header()
```

Comment:
Is there a particular reason to force the signal_streams_header and signal_chan_header calls here? If there are no signals, these two would be expected to be empty lists, as they were before, no?

Reply:
Hmm, I don't remember why I did that. At the moment It does not seem needed. And if changed it does pass the tests... :-)

I guess one might ask whether `_get_signal_streams_header()` should not return an empty list in this case. But I am happy to just change it back for now.